Added Python script for visualizations of expenditure

damionrc opened the pull request:
# PR Template
# Any italic text should be deleted from the final Pull Request text, including this line
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Testing Checklist
- [ ] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from NAMES HERE on the Pull Request

## Notes for PRs 
Below the commit message field you decide whether you are committing your changes to the current branch or a new branch. If the current brain is the main branch (develop), commit to a new branch.

### Filling In CheckBox Example
- [x] Example Filled
- [ ] Example Empty

damionrc committed:
Add Python script for visualizations of expenditure

PradiptaDutta63 reviewed (changes requested):
Could you also add an input for user id for selection. And add a comment to inform the present scenario for user selection for others to know when they use the code. Thanks.

damionrc committed:
Update Visualisations of expenditure.py

Made relevant input for user ID and added following comment to showcase selection

Signed-off-by: damionrc <151696815+damionrc@users.noreply.github.com>
